react/components/ProductSpecifications/Wrapper.js

```diff
@@ -0,0 +1,159 @@
+import * as React from 'react'
+import { ProductContext } from 'vtex.product-context'
+import { path, isEmpty, prop, propOr } from 'ramda'
+
+import ProductSpecifications from './index';
+
+const ProductSpecificationsWrapper = ({ specificationsDefault, ...props }) => {
+  const valuesFromContext = React.useContext(ProductContext)
+  const showSpecifications = prop('showSpecifications', specificationsDefault)
+
+  const getSpecifications = () => {
+    const { product } = valuesFromContext
+
+    const choose = path(['specificationGroups', 'specification'], specificationsDefault)
+    const allSpecifications = propOr([], 'properties', product)
+
+    switch (choose) {
+      case 'editor.product-details.product-specifications.chooseDefaultSpecification':
```

Comment:
Don't need to change anything, but you can add a block inside a case so `const`s are only scoped to the block.

example
```js
switch (choose) {
  case 'foo': {
    const bar = 123
  }
}
```